### ISSUE TITLE
Models are out of date with the new pydantic modeling paradigm from pydantic 2.0

### ISSUE THREAD

**Issue by sammaphey** [mongodb-labs/full-stack-fastapi-mongodb#21]
Seems like this is still using the old implementation of `Config` for pydantic. For example:

```python
class RefreshToken(RefreshTokenUpdate):
    class Config:
        from_attributes = True
```

Should now be re-written as:

```
from pydantic import ConfigDict

class RefreshToken(RefreshTokenUpdate):

    model_config = ConfigDict(from_attributes=True)
```

I believe you should be able to use [Bump Pydantic](https://github.com/pydantic/bump-pydantic) to get a lot of these things fixed with minimal work

**Comment by Jibola** [comment]
Thanks for highlighting this issue @sammaphey.

We've gone ahead and captured this in a [JIRA ticket](https://jira.mongodb.org/browse/PYTHON-4198), where you can track its' progress.

Feel free to open a PR related to the issue and we will happily review it any time.

**Comment by Jibola** [comment]
Marking issue as "fixed" with introduction of PR: #29
